copy/copy.go

```diff
@@ -83,10 +86,23 @@ func (d *digestingReader) Read(p []byte) (int, error) {
 type Options struct {
 	RemoveSignatures bool   // Remove any pre-existing signatures. SignBy will still add a new signature.
 	SignBy           string // If non-empty, asks for a signature to be added during the copy, and specifies a key ID, as accepted by signature.NewGPGSigningMechanism().SignDockerManifest(),
+	ReportWriter     io.Writer
```

Comment:
I decided the assumption that io.Writer is enough based on the fact that I'm explicitly not documenting this since it'll need a well done refactor as soon as I can. I assume everyone will pass in os.Stdout for now